cves/CVE-2015-5351.yml

```diff
@@ -109,9 +129,10 @@ unit_tested:
     For the "fix" answer below, check if the fix for the vulnerability involves
     adding or improving an automated test to ensure this doesn't happen again.
     Must be just "true" or "false".
-  answer:
-  code:
-  fix:
+  answer: no unit test found
```

Comment:
It might be a good idea to list the places that you looked for evidence of unit tests in case someone in the future wants to do a bit more research.